src/freertos_drivers/esp32/Esp32Gpio.hxx

```diff
@@ -35,7 +35,7 @@
 #ifndef _DRIVERS_ESP32GPIO_HXX_
 #define _DRIVERS_ESP32GPIO_HXX_
 
-#include "freertos_drivers/arduino/GpioWrapper.hxx"
+#include "freertos_drivers/common/GpioWrapper.hxx"
```

Comment:
should we update `arduino/libify.sh` since it handles copying from common -> arduino? [ref](https://github.com/bakerstu/openmrn/blob/eb8ed8b3d71d12b9123d4f0a3a0467693c612d96/arduino/libify.sh#L212-L219)

Reply:
in libify they are copied to arduino from common.

Reply:
let's track this via an issue to unify the libify bits so we can avoid ifdef etc